update.sh

```diff
@@ -16,7 +16,7 @@ case "$ruby" in
 		version_family="${version:0:3}"
 
 		exts=(tar.gz tar.bz2 tar.xz zip)
-		downloads_url="http://cache.ruby-lang.org/pub/ruby/"
+		downloads_url="http://cache.ruby-lang.org/pub/ruby"
```

Comment:
Actually all `downloads_url` should be without the final `/`, as there is always a `/` added later on.
Could you remove it also in other places?

Reply:
Done!

Reply:
Thanks!